ui/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.4'
+services:
+  ui:
+    build:
+      context: ..
+      dockerfile: ui/Dockerfile
+    ports:
+    - "${PORT}:${PORT}"
```

Comment:
seems that configuration of this parameter is not reflected in the readme file.

Reply:
The `${PORT}` variable is taken from `.env`, and there is a step about creating it from `.env.example` which contains a default PORT of 3001.

I have added a line to readme explaining that the UI will be served on `http://localhost:PORT`, because it is printed out by the react server, but it's not visible when running detached docker.

Will that be enough in this regard?

Reply:
it is OK for me.